src/platforms/node/common/integrations/default-integrations.mdx

```diff
@@ -5,12 +5,18 @@ redirect_from:
   - /platforms/node/default-integrations/
 ---
 
-System integrations are integrations enabled by default that combine into the
+System integrations are integrations enabled by default that hook into the
 standard library or the interpreter itself. They're documented so you can see
-what they do and that they can be disabled if they cause issues. To disable
-system integrations set `defaultIntegrations: false` when calling `init()`.
-To override their settings, provide a new instance with your config
-to `integrations` option. For example: to change fatal error handler
+what they do and know how to disable them if they cause issues. 
+
+To disable _all_ default integrations, set `defaultIntegrations: false` when calling `init()`.
+
+To disable just one integration, provide a filtered list to the `integrations` option when calling `init()`.
```

Comment:
Maybe a bit pedantic, but the example here shows that what has to be provided is a `function ([]integrations) => []integrations`, not a list/array directly.

```suggestion
To disable one or more of the default integrations, pass in a function to the
`integrations` option when calling `init()`. The function should take an array
of default integrations and return a filtered array, excluding the integrations
you do not want to use.
```

Reply:
Good point, thanks.